### PR TITLE
Feat/pileks/docs revamp v3

### PR DESCRIPTION
**Pull request by pileks**
Refactor based on the following Notion page:
https://www.notion.so/polywrap/Docs-Update-June-23-f430b1aa65dc43bb840ceba771cdecc8

### CODE REVIEW

File: sidebars.js [hunk 1]
```diff
@@ -15,162 +15,84 @@
...
         },
         {
           type: 'category',
-          label: 'Create Plugin Wrappers',
+          label: 'Create Plugin Wraps',
           items: [
-            'tutorials/create-plugin-wrappers/create-js-plugin',
-            'tutorials/create-plugin-wrappers/plugin-manifest',
-            'tutorials/create-plugin-wrappers/plugin-architecture'
+            'tutorials/create-plugin-wraps/create-js-plugin',
```
Comment (dOrgJelli) [inline sidebars.js]: Are we still calling these "plugin wraps" or should they just be "plugins"?

File: src/docs-old/reference/cli/commands/build.md
Comment (dOrgJelli) [inline src/docs-old/reference/cli/commands/build.md:1]: We can just get rid of these, no? No sense in having the "docs-old" folder.

File: src/docs/what-is-polywrap.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
+---
+id: what-is-polywrap
+title: What is Polywrap
```
Comment (dOrgJelli) [inline src/docs/what-is-polywrap.md:3]: This seems strange to have when we already have an introduction

File: sidebars.js (hunk 1 above)
Comment (pileks) [reply sidebars.js]: Will be called Plugins 😄

File: src/docs-old/reference/cli/commands/build.md
Comment (pileks) [reply src/docs-old/reference/cli/commands/build.md:1]: Yes, we can!

File: src/docs/what-is-polywrap.md (hunk 2 above)
Comment (pileks) [reply src/docs/what-is-polywrap.md:3]: I kept it as such, as it's a copy of the more detailed explanation from our Blog, and it was the plan within the Notion.

I'd wager we can remove this? Or maybe turn it into a "Web3 use-case" kind of doc?

**Review by kev-ngo — APPROVED** [review]
This looks much cleaner, and is a great starting point for making adjustments over the following weeks.

**Pull request merged**